LoggingWrapper object has no attribute ''debug_exception''' when trying to sync feeds

**Is this a BUG REPORT or a FEATURE REQUEST?** (choose one): BUG REPORT

**Version of Anchore Engine and Anchore CLI if applicable**:
anchore-cli, version 0.6.1

** What happened? **

I have set up AnchoreE Community edition in a Kubernetes 1.15.5 cluster, using the helm chart.  I have most things up and running.About 12 hours ago, I did:

```
anchore-cli --version
anchore-cli, version 0.6.1
```

 anchore-cli  system feeds syncWARNING: This operation should not normally need to be performed except when the anchore-engine operator is certain that it is required - the operation will take a long time (hours) to complete, and there may be an impact on anchore-engine performance during the re-sync/flush.Really perform a manual feed data sync/flush? (y/N)y

12 hours later, I did:

```
anchore-cli  system feeds list

Feed                   Group                  LastSync        RecordCount
nvdv2                  nvdv2:cves             pending         0
vulnerabilities        alpine:3.10            pending         0
vulnerabilities        alpine:3.11            pending         0
vulnerabilities        alpine:3.3             pending         0
vulnerabilities        alpine:3.4             pending         0
vulnerabilities        alpine:3.5             pending         0
vulnerabilities        alpine:3.6             pending         0
vulnerabilities        alpine:3.7             pending         0
vulnerabilities        alpine:3.8             pending         0
vulnerabilities        alpine:3.9             pending         0
vulnerabilities        amzn:2                 pending         0
vulnerabilities        centos:5               pending         0
vulnerabilities        centos:6               pending         0
vulnerabilities        centos:7               pending         0
vulnerabilities        centos:8               pending         0
vulnerabilities        debian:10              pending         0
vulnerabilities        debian:11              pending         0
vulnerabilities        debian:7               pending         0
vulnerabilities        debian:8               pending         0
vulnerabilities        debian:9               pending         0
vulnerabilities        debian:unstable        pending         0
vulnerabilities        ol:5                   pending         0
vulnerabilities        ol:6                   pending         0
vulnerabilities        ol:7                   pending         0
vulnerabilities        ol:8                   pending         0
vulnerabilities        ubuntu:12.04           pending         0
vulnerabilities        ubuntu:12.10           pending         0
vulnerabilities        ubuntu:13.04           pending         0
vulnerabilities        ubuntu:14.04           pending         0
vulnerabilities        ubuntu:14.10           pending         0
vulnerabilities        ubuntu:15.04           pending         0
vulnerabilities        ubuntu:15.10           pending         0
vulnerabilities        ubuntu:16.04           pending         0
vulnerabilities        ubuntu:16.10           pending         0
vulnerabilities        ubuntu:17.04           pending         0
vulnerabilities        ubuntu:17.10           pending         0
vulnerabilities        ubuntu:18.04           pending         0
vulnerabilities        ubuntu:18.10           pending         0
vulnerabilities        ubuntu:19.04           pending         0
vulnerabilities        ubuntu:19.10           pending         0
```

I ran this command:

```
anchore-cli event list

Timestamp                          Level        Event                                  Resource                               ID
2020-03-05T16:22:37.411806Z        info         system.feeds.sync.completed            None                                   8243077a04554835bb0cda92f6a5008a
2020-03-05T16:22:37.102255Z        error        system.feeds.sync.feed_failed          nvdv2                                  be7771bfb0054bc79a0ffb8eeb8c033f
2020-03-05T16:22:36.981029Z        error        system.feeds.sync.group_failed         nvdv2/nvdv2:cves                       55dac8443bee451ba57290b241e25583
2020-03-05T16:22:36.856560Z        info         system.feeds.sync.group_started        nvdv2/nvdv2:cves                       753428a656364e0e9ff5424f1e29b043
2020-03-05T16:22:36.729406Z        info         system.feeds.sync.feed_started         nvdv2                                  5b725d81e45540aeb9f9a4cd75d5f589
```

and this:

```
anchore-cli event get 55dac8443bee451ba57290b241e25583
details:
  cause: '''LoggingWrapper'' object has no attribute ''debug_exception'''
level: error
message: Feed group sync failed
resource:
  id: nvdv2/nvdv2:cves
  type: feed_group
  user_id: admin
source:
  base_url: http://anchore-anchore-engine-policy:8087
  hostid: anchore-anchore-engine-policy-56767545df-mrghz
  request_id: null
  servicename: policy_engine
timestamp: '2020-03-05T16:22:36.981029Z'
type: system.feeds.sync.group_failed
```

@zhill I found this in the `anchore-anchore-engine-policy` pod


```
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/error()] [ERROR] Error syncing vulnerabilities/alpine:3.10 (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Event: {"type": "system.feeds.sync.group_failed", "level": "error", "message": "Feed group sync failed", "details": {"cause": "'LoggingWrapper' object has no attribute 'debug_exception'"}, "timestamp": "2020-03-05T04:21:10.112922", "resource": {"user_id": "admin", "type": "feed_group", "id": "vulnerabilities/alpine:3.10"}, "source": {"request_id": null, "servicename": "policy_engine", "hostid": "anchore-anchore-engine-policy-56767545df-mrghz", "base_url": "http://anchore-anchore-engine-policy:8087"}} (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] Traceback (most recent call last):^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/anchore_engine/services/policy_engine/engine/feeds/sync.py", line 333, in sync^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-]     feed_data_repo.teardown()^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] AttributeError: 'NoneType' object has no attribute 'teardown'^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/exception()] [ERROR] Could not cleanup download repo due to error^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Event: {"type": "system.feeds.sync.group_started", "level": "info", "message": "Feed group sync started", "details": {}, "timestamp": "2020-03-05T04:21:10.174506", "resource": {"user_id": "admin", "type": "feed_group", "id": "vulnerabilities/alpine:3.11"}, "source": {"request_id": null, "servicename": "policy_engine", "hostid": "anchore-anchore-engine-policy-56767545df-mrghz", "base_url": "http://anchore-anchore-engine-policy:8087"}} (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Beginning feed data fetch (feed=vulnerabilities, group=alpine:3.11, operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/error()] [ERROR] Error syncing vulnerabilities/alpine:3.11 (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Event: {"type": "system.feeds.sync.group_failed", "level": "error", "message": "Feed group sync failed", "details": {"cause": "'LoggingWrapper' object has no attribute 'debug_exception'"}, "timestamp": "2020-03-05T04:21:10.235941", "resource": {"user_id": "admin", "type": "feed_group", "id": "vulnerabilities/alpine:3.11"}, "source": {"request_id": null, "servicename": "policy_engine", "hostid": "anchore-anchore-engine-policy-56767545df-mrghz", "base_url": "http://anchore-anchore-engine-policy:8087"}} (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] Traceback (most recent call last):^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/anchore_engine/services/policy_engine/engine/feeds/sync.py", line 333, in sync^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-]     feed_data_repo.teardown()^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] AttributeError: 'NoneType' object has no attribute 'teardown'^M
```
[service:policy-engine] 2020-03-05 04:21:08+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Initialized feed to sync: nvdv2 (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:09+0000 [-] [Thread-11] [anchore_engine.clients.services.internal/dispatch()] [ERROR] Failed client call to service catalog for url: http://anchore-anchore-engine-catalog:8082/v1/events. Response: {'httpcode': 500, 'anchore_error_raw': "HTTPConnectionPool(host='anchore-anchore-engine-catalog', port=8082): Max retries exceeded with url: /v1/events (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f26baa2ee80>: Failed to establish a new connection: [Errno 111] Connection refused',))", 'anchore_error_json': {}}^M
[service:policy-engine] 2020-03-05 04:21:09+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/warn()] [WARN] Error adding feed start event: failed post url=http://anchore-anchore-engine-catalog:8082/v1/events^M
[service:policy-engine] 2020-03-05 04:21:09+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Event: {"type": "system.feeds.sync.feed_started", "level": "info", "message": "Feed sync started", "details": {}, "timestamp": "2020-03-05T04:21:08.019756", "resource": {"user_id": "admin", "type": "feed", "id": "vulnerabilities"}, "source": {"request_id": null, "servicename": "policy_engine", "hostid": "anchore-anchore-engine-policy-56767545df-mrghz", "base_url": "http://anchore-anchore-engine-policy:8087"}} (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Event: {"type": "system.feeds.sync.group_started", "level": "info", "message": "Feed group sync started", "details": {}, "timestamp": "2020-03-05T04:21:09.030841", "resource": {"user_id": "admin", "type": "feed_group", "id": "vulnerabilities/alpine:3.10"}, "source": {"request_id": null, "servicename": "policy_engine", "hostid": "anchore-anchore-engine-policy-56767545df-mrghz", "base_url": "http://anchore-anchore-engine-policy:8087"}} (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Beginning feed data fetch (feed=vulnerabilities, group=alpine:3.10, operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/error()] [ERROR] Error syncing vulnerabilities/alpine:3.10 (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
[service:policy-engine] 2020-03-05 04:21:10+0000 [-] [Thread-11] [anchore_engine.services.policy_engine.engine.logs/info()] [INFO] Event: {"type": "system.feeds.sync.group_failed", "level": "error", "message": "Feed group sync failed", "details": {"cause": "'LoggingWrapper' object has no attribute 'debug_exception'"}, "timestamp": "2020-03-05T04:21:10.112922", "resource": {"user_id": "admin", "type": "feed_group", "id": "vulnerabilities/alpine:3.10"}, "source": {"request_id": null, "servicename": "policy_engine", "hostid": "anchore-anchore-engine-policy-56767545df-mrghz", "base_url": "http://anchore-anchore-engine-policy:8087"}} (operation_id=c482e9dc1a124d5d91d347430d2d824d)^M
```

--- a/anchore_engine/services/policy_engine/engine/feeds/download.py
+++ b/anchore_engine/services/policy_engine/engine/feeds/download.py
@@ -9,8 +9,7 @@ from anchore_engine.utils import mapped_parser_item_iterator
 from anchore_engine.services.policy_engine.engine.feeds import IFeedSource
 from anchore_engine.services.policy_engine.engine.feeds.schemas import LocalFeedDataRepoMetadata, DownloadOperationConfiguration, DownloadOperationResult, GroupDownloadOperationConfiguration, FeedAPIGroupRecord, FeedAPIRecord, GroupDownloadResult
 
-from anchore_engine.services.policy_engine.engine.logs import get_logger
-logger = get_logger()
+from anchore_engine.subsys import logger
 
 from anchore_engine.utils import ensure_bytes, timer
 

